doc/changes/latest.rst

```diff
@@ -29,6 +29,7 @@ Fixes
   Now if band-pass elements are equal :func:`~nilearn.signal.butterworth` returns an unfiltered signal with a warning (:gh:`3293` by `Yasmin Mzayek`_).
 - The parameter ``alpha`` is now correctly passed to :func:`~plotting.plot_glass_brain` in :func:`~plotting.plot_connectome` (:gh:`3306` by `Koen Helwegen`_).
 - Fix plotting of background image in :func:`~nilearn.plotting.view_img` when the background is not the MNI template (:gh:`3312` by `Jerome Dockes`_).
+- Fix the typographic error on the page https://nilearn.github.io/dev/auto_examples/04_glm_first_level/plot_adhd_dmn.html#sphx-glr-auto-examples-04-glm-first-level-plot-adhd-dmn-py (:gh:`3336` by `Sachin Patalasingh`_)
```

Comment:
I changed the link to use sphinx cross-referencing and the github link should be to this pull request rather than the issue number

```suggestion
- Fix the typographic error on the page :ref:`sphx-glr-auto-examples-04-glm-first-level-plot-adhd-dmn.py` (:gh:`3337` by `Sachin Patalasingh`_).
```

Reply:
I am sorry as I didnot know what things I had to add in whats new entry. Thank you so much @ymzayek. This looks great. I am commiting your suggestions. Thanks again.